### DISCUSSION TITLE
Using quotations returns more results targeting same scope

### DISCUSSION THREAD

**Discussion by Joclaim** [microsoft-search/pnp-modern-search#2601] [Q&A] [answered]
**Problem**
Search returns a different number of results based on the use of quotations or not while targeting the same scope.
•	When we search for project manager there are 2 results returned
•	When we search for “project manager” there are 4 results returned
Obviously, this would make sense in many circumstances but in our setup, we are unable to explain this behavior or find a workaround to create consistency.

**Setup**
We are targeting a SharePoint list that have a few columns but the most important once for this troubleshooting are a people field and a Roles field (Lookup).
1.	The query in the results web part is configured as follow:
RefinableString104:{searchTerms} AND Path:https://domain.sharepoint.com/sites/quality/Lists/Employee 
Where the refinablestring104 is the roles field 
2.	The Refinement filters are empty
3.	We using the People Layout and have added the Role as Secondary text in the Manage people field.

**Reproducing**
In the PnP search box we search for project manager and we receive 2 results, the Role is displayed and for 2 people we can read: Role: Project Manager. 
In the PnP search box we search for “project manager” and we receive 4 results, the Role is displayed and for 4 people we can read: Role: Project Manager. 
In our thought process, we are querying the RefinableString104 field, scoping out other fields. Using no quotes would rather give more results then less.

**Question**
Could you explain this behavior or guide us through a solution that would create consistency in the returned result.

**Comment by wobba** [comment, marked as answer]
You should change your query template to `RefinableString104:({searchTerms})` to ensure multi terms are queried towards the property, and not only the first one.

**Reply by Joclaim** [reply]
Hi Wobba,

That indeed results in 6 results when searching project manager and in 4 when searching "project manager". Which makes sense as in the first attempt it searches for the word project separately.

Thank you for your help, much appreciated.
